formats/json/commonTest/src/kotlinx/serialization/features/JsonAlternativeNamesTest.kt

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2017-2021 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+@file:Suppress("ReplaceArrayOfWithLiteral") // https://youtrack.jetbrains.com/issue/KT-22578
+
+package kotlinx.serialization.features
+
+import kotlinx.serialization.*
+import kotlinx.serialization.json.*
+import kotlinx.serialization.test.*
+import kotlin.test.*
+
+@Serializable
+data class WithNames(@JsonNames(arrayOf("foo", "_foo")) val data: String)
```

Comment:
Code style: classes are not within tests and tests not started with `test` prefix